certbot_s3front/installer.py

```diff
@@ -60,6 +60,21 @@ def deploy_cert(self, domain, cert_path, key_path, chain_path, fullchain_path):
         body = open(cert_path).read()
         key = open(key_path).read()
         chain = open(chain_path).read()
+        
+        # Delete old cert
+        try:
+            client.delete_server_certificate(
+                ServerCertificateName=name
+            )
+        except botocore.exceptions.ClientError as e:
+            logger.error(e)
+
+        # Rename cert to the new one
+        client.update_server_certificate(
+            ServerCertificateName=name + '-new',
+            NewServerCertificateName=name
+        )
```

Comment:
@dlapiduz I believe this fix needs rolling back: note that the rename now comes before the "-new" cert is even uploaded, which doesn't make sense to me.

See also my comment just now on #45 - I believe the problem we're trying to fix here was due to a bug in AWS, not in certbot-s3front.